test/tests/pbs_smoketest.py

```diff
@@ -150,21 +147,27 @@ def test_standing_reservation_stale_nodes(self):
              'reserve_end': time_now + 14, 
              'Resource_List.place': 'free'}
         r = Reservation(TEST_USER, a)
-        #Submitting recurring reservation to check the resv status for one of the vnode being stale
+        #Submitting recurring reservation to check the resv status 
+        #for one of the vnode being stale
         rid = self.server.submit(r)
         #To check the standing resv once confirmed.
         a = {'reserve_state': (MATCH_RE, "RESV_CONFIRMED|2")}
-        #To check the standing resv for Degraded status when one of the vnode becomes down
+        #To check the standing resv for Degraded status 
+        #when one of the vnode becomes down
         a2 = {'reserve_state': (MATCH_RE, "RESV_DEGRADED|10")}
         self.server.expect(RESV, a, id=rid)
         self.server.status(RESV, 'resv_nodes', id=rid)
         resv_node = self.server.reservations[rid].get_vnodes()[0]
-        #To check the status of standing resv in next cycle and also check the down node doesn't exist in resv_nodes.
-        a = {'reserve_state': (MATCH_RE, "RESV_CONFIRMED|2"),'resv_nodes': (MATCH_RE, '^((?!' + re.escape(resv_node) + ').)*$')}
+        #To check the status of standing resv in next cycle and also 
+        #check the down node doesn't exist in resv_nodes.
+        a = {'reserve_state': (MATCH_RE, "RESV_CONFIRMED|2"),
+             'resv_nodes': 
+             (MATCH_RE, '^((?!' + re.escape(resv_node) + ').)*$')}
         b = {'state': 'down'}
         self.server.manager(MGR_CMD_SET, NODE, b, id=resv_node)
         self.server.expect(RESV, a2, id=rid)
-        self.logger.info('testinfo: waiting for recurring reservation, while expecting status')
+        self.logger.info('testinfo: waiting for recurring reservation,' + 
+                                                'while expecting status')
```

Comment:
No need for the '+'.  Two string literals next to each other will be concatenated together.